### PR TITLE
feat(i18n): add Korean translations

### PR DESCRIPTION
**Pull request by dodok8**
Add Korean for @valibot/i18n

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

**Comment by dodok8** [comment]
I asked my fellow developers for a review. However, this weekend is one of  the biggest holiday of Korea, Lunar New Year. So it might take some time.

**Comment by fabian-hiller** [comment]
Thank you and no worries!

**Review by baekrang256 — APPROVED** [review]
Seems to be done well. I think it's good to go 😄

**Comment by fabian-hiller** [comment]
Thank you for your review!

File: packages/i18n/src/kr.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    ipv6: (issue) => `유효하지 않은 IPv6: ${issue.received} 을(를) 받았습니다.`,
+    isoDate: (issue) => `유효하지 않은 날짜: ${issue.received} 을(를) 받았습니다.`,
+    isoDateTime: (issue) =>
+      `유효하지 않은 날짜-시각: ${issue.received} 을(를) 받았습니다.`,
+    isoTime: (issue) => `유효하지 않은 시각: ${issue.received} 을(를) 받았습니다.`,
+    isoTimeSecond: (issue) =>
+      `유효하지 않은 시각-초: ${issue.received} 을(를) 받았습니다.`,
+    isoTimestamp: (issue) =>
+      `유효하지 않은 타임스탬프: ${issue.received} 을(를) 받았습니다.`,
+    isoWeek: (issue) => `유효하지 않은 요일: ${issue.received} 을(를) 받았습니다.`,
```
Comment (2paperstar) [inline packages/i18n/src/kr.ts:45]: korean translation means "day of week" but this library intends "iso week" (yyyy-Www)
reference: https://github.com/fabian-hiller/valibot/blob/main/library/src/validations/isoWeek/isoWeek.ts

Comment (2paperstar) [reply packages/i18n/src/kr.ts:45]: I think `유효하지 않은 주` is better

File: packages/i18n/src/kr.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    emoji: (issue) => `유효하지 않은 이모지: ${issue.received} 을(를) 받았습니다.`,
+    endsWith: (issue) =>
+      `유효하지 않은 끝: ${issue.expected} 을(를) 예상했으나 ${issue.received} 을(를) 받았습니다.`,
+    equal: (issue) =>
+      `유효하지 않은 값: ${issue.expected} 을(를) 예상했으나 ${issue.received} 을(를) 받았습니다.`,
+    excludes: (issue) =>
+      `유효하지 않은 컨텐츠: ${issue.expected} 을(를) 예상했으나 ${issue.received} 을(를) 받았습니다.`,
+    finite: (issue) => `유효하지 않은 유한 숫자: ${issue.received} 을(를) 받았습니다.`,
+    hash: (issue) => `유효하지 않은 해쉬: ${issue.received} 을(를) 받았습니다.`,
+    hexColor: (issue) => `유효하지 않은 hex 색상: ${issue.received} 을(를) 받았습니다.`,
```
Comment (2paperstar) [inline packages/i18n/src/kr.ts:27]: hex color is most translated as `헥스 코드` than `hex 색상`, but `hex 색상` is also good.

**Review by 2paperstar — CHANGES_REQUESTED** [review]

**Review by 2paperstar — APPROVED** [review]
it seems good 👍

**Comment by fabian-hiller** [comment]
Thank you! I plan to merge this PR today.

**Comment by fabian-hiller** [comment]
Would it be wrong to remove the dot at the end of each sentence? If so, I would merge the PR that way. If not, I would remove the dot to be consistent with the other translations.

**Comment by 2paperstar** [comment]
> Would it be wrong to remove the dot at the end of each sentence? If so, I would merge the PR that way. If not, I would remove the dot to be consistent with the other translations.

No. It’s ok to remove them.

**Pull request merged**